design/high-level-reactivity.md

```diff
@@ -223,6 +275,16 @@ A first-class developer friendly backend data driver will need to:
 
 While the ideal driver would emit exact deps and invalidations automatically for all queries, this is not likely to be practical for an arbitrary datastore. In practice, drivers will likely fall back to broader deps and invalidations, and tooling will help developers identify places where these invalidations can be optimized.  The developer can then rework their query or emit manual invalidations as needed.
 
+// How about the design of the Invalidation Server. Here's my ideas:
+ * It would be great if it's stateless, so if we need more load we can just scale it.
+ * If not, we could shard it scale out. 
+ * This is not a thing we need to do now.
+ * But, we should plan for it.
+ 
+// Are we going to write it in Node.js
+
+I assume drivers need to do something in the invalidation later too. (Data fetching). So it's needed to do it in Node.js?
```

Comment:
I mean how do we fetch data for the invalidations. 
Ah ha. Got it. You just need to refetch it from the client.
Invalidation server doesn't know about data.